opensrp-chw-core/src/main/java/org/smartregister/chw/core/dao/ChildDao.java

```diff
@@ -132,29 +131,27 @@ public static boolean isChild(String baseEntityID) {
         return res.get(0) > 0;
     }
 
-    public static String getBaseEntityID(String identifierType, String id) {
-        String selection = "select base_entity_id from ec_child where ? = ?";
-        String[] selectionArgs = new String[]{identifierType, id};
```

Comment:
Why was this removed, is this not being used by others.

Reply:
This method is added again.

Reply:
This method is added again.